File: proxy-server/Server.Domain/Application/RouterChild.cs
using Newtonsoft.Json;

namespace Server.Domain.Application;

public class RouterChild
{
  [JsonProperty("identity")]
  public string Identity { get; set; } = string.Empty;

  [JsonProperty("isDefault")]
  public bool IsDefault { get; set; }

  [JsonProperty("longName")]
  public string JsonPropertyLongName { get; set; } = string.Empty;

  [JsonProperty("service")]
  public string Service { get; set; } = string.Empty;

  [JsonProperty("shortName")]
  public string ShortName { get; set; } = string.Empty;
}

File: blip-cli/chat/src/types/router/router_child.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize, Debug)]
pub struct RouterChild {
    #[serde(rename = "identity")]
    pub identity: String,

    #[serde(rename = "isDefault")]
    pub is_default: bool,

    #[serde(rename = "longName")]
    pub long_name: String,

    #[serde(rename = "service")]
    pub service: String,

    #[serde(rename = "shortName")]
    pub short_name: String,
}